src/scripts/main.js

```diff
@@ -2,7 +2,32 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const dashboard = document.querySelector('.dashboard');
 
-// write your code here
+for (const person of people) {
+  // eslint-disable-next-line no-shadow
+  const { name, sex, born, died } = person;
```

Comment:
```suggestion
  const { 
    name: personName, 
    sex, 
    born, 
    died
  } = person;
```
Please format your code to make it cleaner.
Also, you can rename variables as follow to avoid the error